.ncurc.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    // Whitelist all for checking besides `peer` which indicates
```

Comment:
I didn’t know about `npm-check-updates`. I always used `npx david`, but this tool looks more straight forward. Thanks 👍.